cmd/manager/main.go

```diff
@@ -286,7 +286,12 @@ func Command() *cobra.Command {
 	cmd.Flags().Int(
 		operator.MetricsPortFlag,
 		DefaultMetricPort,
-		"Port to use for exposing metrics in the Prometheus format (set 0 to disable)",
+		"Port to use for exposing metrics in the Prometheus format. (set 0 to disable)",
+	)
+	cmd.Flags().String(
+		operator.MetricsHostFlag,
+		"0.0.0.0",
+		fmt.Sprintf("The host to which the operator should bind to serve metrics in the Prometheus format. Will be combined with %s.", operator.MetricsPortFlag),
```

Comment:
Why are we adding an option for the metrics host ? Just got my understanding.

Reply:
I believe there was a previous discussion in this PR. It's to allow the user to lock down the ECK operators metrics port to force communication through the proxy when using the manifests (helm handles it for the user)